pkg/minishift/network/settings.go

```diff
@@ -43,6 +58,18 @@ type NetworkSettings struct {
 	Gateway   string
 	DNS1      string
 	DNS2      string
+	UseDHCP   bool
+	Disabled  bool
+}
+
+func checkSupportForAddressAssignment() bool {
+	if minishiftConfig.InstanceConfig.IsRHELBased &&
+		minishiftConfig.InstanceConfig.SupportsNetworkAssignment {
+		return true
+	} else {
+		atexit.ExitWithMessage(1, configureNetworkNotSupportedMessage)
+	}
+	return false
```

Comment:
Is this function used anywhere? I can not find any reference?

Reply:
part of #1457. Will add this to the commit...